docs/optimised_config.md

```diff
@@ -40,6 +40,8 @@ We performed similar experiments for CPU (100 requests from 100 concurrent worke
 We also found similar results for CPU as shown in the above plots i.e throughput is highest when there are 8 workers(which equals number of vCPUs in c5.2xlarge)
 **Based on the results, we recommend setting number of Gunicorn workers equal to number of vCPUs present in the instance.**
 
+**Note:** The higher latencies seen during the load tests on CPU instance is because the c5.2xlarge CPU instance has 1/4th the total number of vCPUs as compared the p3.8xlarge GPU instance and both were serving the same number of incoming request. Apart from it, the GPU also shares some workload of CPUs avoiding any backlogs and hence better latency numbers. We had seen significantly lesser latencies when the rate of requests coming into c5.2xlarge instance were reduced by 1/4th.
```

Comment:
The higher latencies were seen during the load tests on CPU instance was because the c5.2xlarge CPU instance has 1/4th the total number of vCPUs as compared to the p3.8xlarge GPU instance and both were serving the same number of incoming request. Apart from it, the GPU also shares some workload of CPUs reducing the request backlogs. We had seen significantly lesser latencies when the rate of requests coming into c5.2xlarge instance was reduced by 1/4th